src/modules/core/components/TokenEditDialog/TokenEditDialog.tsx

```diff
@@ -46,28 +47,36 @@ const MSG = defineMessages({
 });
 
 interface Props {
-  addTokenFn: (address: Address) => Promise<any>;
+  updateTokens: (addresses: Address[]) => Promise<any>;
   cancel: () => void;
   close: () => void;
+  // Colony tokens
   tokens: AnyToken[];
+  // Token list from json file. Not supported on local env
+  tokensList?: AnyToken[];
+  // Colony native token addresss
+  nativeTokenAddress?: Address;
 }
 
 interface FormValues {
   tokenAddress: Address;
   description?: string;
+  tokenAddresses: Address[];
 }
 
 const validationSchema = yup.object({
-  tokenAddress: yup
-    .string()
-    .address()
-    // @todo validate against entering a duplicate address
-    .required(),
+  tokenAddress: yup.string().address(),
   tokenSymbol: yup.string().max(6),
```

Comment:
i mean, it was already here, i didin't add it in this PR. I can remove it

Reply:
Just because something is wrong, and you notice it, please remove it. Doesn't matter if it was added by your PR or not.

This will help us keep a more maintainable codebase for longer.

Remember, the codebase is owned by all of us! :tada: